fabric-renderer-registries-v1/src/main/java/net/fabricmc/fabric/api/client/rendereregistry/v1/item/ItemOverlayRendererRegistry.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.rendereregistry.v1.item;
+
+import net.minecraft.item.Item;
+
+import net.fabricmc.fabric.impl.client.renderer.registry.item.ItemOverlayExtensions;
+
+public final class ItemOverlayRendererRegistry {
+	private ItemOverlayRendererRegistry() { }
+
+	public static CountLabelProperties getCountLabelProperties(Item item) {
```

Comment:
All of these should be ItemConvertible instead of Item.

This is so people can pipe a block into the method.

Reply:
time to regex this shit up

Reply:
Rofl in the method just call `asItem` and null check it lol

Reply:
oh oops i forgot the null check part

Reply:
so in getters, if `ItemConvertible.asItem()` returns `null`, should i also return `null` or throw an `IllegalArgumentException`

Reply:
I usually just Objects.requireNonNull the item convertible instance and then the result of `asItem`

Reply:
1acbb44 :)